linux_sql/scripts/host_info.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+psql_host=$1
+psql_port=$2
+db_name=$3
+psql_user=$4
+psql_password=$5
+
+#check # of args
+if [ "$#" -ne 5 ]; then
+    echo "Illegal number of parameters"
```

Comment:
It's good UX practice to return the proper formatting or number of parameters to the user with these sorts of errors.